### ISSUE TITLE
How to handle screen rotation

### ISSUE THREAD

**Issue by MarioUnlam** [RepairShopr/react-native-signature-capture#225]
I'm having problems with this plugin when the screen changes orientation. The canvas size seems to be fixed on mount, and I can't find a way to resize it. If the screen is initially in portrait mode, and then it changes to landscape, the canvas is smaller than its container. If it starts with landscape and then changes to portrait, the canvas is larger than the container (and the image is cut). Is there a way to "refresh" the size of the canvas? So I can refresh it when the orientation changes, keeping the aspect ratio.

**Comment by yestay90** [comment]
@MarioUnlam , hello, I am having the same problem. Have you found the solution?

**Comment by lededje** [comment]
@yestay90 @MarioUnlam I had the same issue; I used an orientation library to check to see which orientation it was in and either only display the signature when it is in portrait (and display an image prompting the user to rotate the device) or set the key of the signature to change each time the orientation does (note doing this will reset the signature component including anything drawn up to that point).

**Comment by MarioUnlam** [comment]
@lededje Yeah I thought about adding a key attribute to remount the signature component, but that makes the signature reset, which could be a problem if the user rotates the device to landscape only to sign, and then goes back to portrait to press the save button. We left landscape mode disabled for now in our app, but soon we will have to implement it. Another approach that could work is making the container dimensions fixed, trying to make it look good in both orientations.
